src/Makefile.am

```diff
@@ -41,6 +41,7 @@ if LINUX
 else
 if WINDOWS
   mednaffe_SOURCES+= widgets/joystick_windows.c
+  mednaffe_SOURCES+= win32util.c win32util.h
```

Comment:
Here you need add the files to the previous one:
`mednaffe_SOURCES+= win32util.c win32util.h widgets/joystick_windows.c`